File: Programmers/JAVA/lv1/lv1_2016년.java
https://programmers.co.kr/learn/courses/30/lessons/12901?language=java

class Solution {
    public String solution(int a, int b) {
        String answer = "";
        String[] days = {"SUN","MON","TUE","WED","THU","FRI","SAT"};
        int tmp = 0;
        
        for(int i = 1; i < a; i++){
            if(i == 2)
                tmp += 29;
            else if(i == 4 || i == 6 || i == 9 || i == 11)
                tmp += 30;
            else
                tmp += 31;
        }
        tmp += b - 1;
        answer = days[(tmp+5)%7];   //배열 안에서 계산 시 런타임 에러
        
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_2016년.cs
https://programmers.co.kr/learn/courses/30/lessons/12901?language=csharp

using System;

public class Solution
{
    public string solution(int a, int b)
    {
        string answer = "";

        DateTime date = new DateTime(2016, a, b);
        //Console.WriteLine(date);  // 5/24/2016 12:00:00 AM
        answer = date.DayOfWeek.ToString();
        //Console.WriteLine(answer);    // 	Tuesday
        answer = answer.Substring(0, 3).ToUpper();

        return answer;
    }
}